### ISSUE TITLE
~48mb xlsx = 2gb ram

### ISSUE THREAD

**Issue by RusJJ** [troldal/OpenXLSX#89]
So, as you can see, ~48mb xlsx document uses 2gb of RAM. Any way to fix that? Kinda a lot, since Excel uses A LOT less memory.

**Comment by troldal** [comment]
The file size of 48 MB is the compressed (.zip) size. If you uncompress it, my guess is that it will be 400-500 MB. Using 2GB (4-5x the size of the raw data) of ram for that is actually not unreasonable.

Manipulating large excel spreadsheets is a tradeoff between speed and memory consumption. If you read the [README](https://github.com/troldal/OpenXLSX#memory-usage) file, you will se that I have adressed excactly that, and the focus has always been on speed...not memory consumption.

So, in short...it's not a bug, so it doesn't need to be fixed.

Also, comparing the library to MS Excel is unfair. Microsoft has dozens of developers for developing Excel, so for sure they have found ways to optimize memory usage. That being said, they have the same tradeoff, so if you try to manipulate a large spreadsheet from Excel VBA, it may ule less ram, but will be much slower.

**Comment by RusJJ** [comment]
Thanks for the answer! I just removed completely useless data from xlsx and now it's 10x lighter.
